aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter-stateless/README.md

```diff
@@ -32,27 +32,30 @@ For the test SPA provided with this example you should create the following role
       "allowedMemberTypes": [
         "User"
       ],
+      "description": "Full admin access",
       "displayName": "Admin",
-      "id": "2fa848d0-8054-4e11-8c73-7af5f1171001",
+      "id": "********-****-****-****-************",
```

Comment:
I suggest you keep the original value;
the value is just UUID as I know, if the user just want to follow the readme to set up the environment,
it's better for users to just copy and paste.